awx/ui_next/src/screens/Inventory/InventoryDetail/InventoryDetail.jsx

```diff
@@ -76,9 +76,11 @@ function InventoryDetail({ inventory, i18n }) {
           value={
             <ChipGroup numChips={5}>
               {instanceGroups.map(ig => (
-                <Chip key={ig.id} isReadOnly>
-                  {ig.name}
-                </Chip>
+                <div css="padding-right: 5px" key={ig.id} isReadOnly>
+                  <Link to={`/instance_groups/${ig.id}/details`}>
+                    <Label>{ig.name}</Label>
```

Comment:
I don't think we should use the Label component within the ChipGroup component. A ChipGroup should contain Chips. Since this is not how either component was intended to be used, I think we need to come up with a different solution.

Reply:
on second thought, thinking about like this...we'd rather not have to do the work twice, so maybe we should hold until they comment on this issue

Reply:
based on https://github.com/patternfly/patternfly-design/issues/751 it seems that they are working on _something_ to help us solve this problem. I'm ok with waiting to see what PF does as this issue isn't high priority right now for us.  If we do have to roll our own think it isn't real complicated.